src/containers/maps/MapVisuals.jsx

```diff
@@ -0,0 +1,171 @@
+/*
+Map Explorer - Visuals
+======================
+
+********************************************************************************
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Actions } from 'jumpstate';
+
+import L from 'leaflet';
+
+import {
+  MAPEXPLORER_INITIAL_STATE, MAPEXPLORER_PROPTYPES
+} from '../../ducks/mapexplorer';
+
+//Arbitrary values for a default map marker.
+//TODO / QUESTION: make this customisable?
+const DEFAULT_MARKER = {
```

Comment:
Sigh, this is definitely something I need to revisit if I want to make the MapExplorer usable for non-Darien, non-Gorongosa projects.